web/turkey/survey/static/survey/js/auditors/mouse_movement_total.js

```diff
@@ -0,0 +1,26 @@
+var AuditorMouseMovementTotal = {
+    mouse_movement_total: 0,
+    previous_position: 0,
+    log_mousemove_event: function (e) {
+            this.mouse_movement_total += 1;
+        }
+    },
+    submit_callable: function () {
+        return {
+            'amount': this.mouse_movement_total
+        };
+    }
+};
+
+var auditor_mouse_movement_total = Object.create(AuditorMouseMovementTotal);
+
+$(document).ready(function() {
+    $(window).mousemove(auditor_mouse_movement_total
```

Comment:
I was unsure of how exactly or precisely we were wanting to analyze mouse movements. I'll take another look.